io.sloeber.product.sdk/arduino.product

```diff
@@ -280,14 +280,15 @@ United States, other countries, or both.
    </configurations>
 
    <repositories>
-      <repository location="http://download.eclipse.org/nebula/releases/latest/" enabled="true" />
-      <repository location="http://eclipse.baeyens.it/update/V4/nightly/" enabled="false" />
-      <repository location="http://eclipse.baeyens.it/update/V4/stable/" enabled="true" />
-      <repository location="http://gnu-mcu-eclipse.netlify.com/v4-neon-updates/" enabled="true" />
-      <repository location="http://download.eclipse.org/releases/latest/" enabled="true" />
+      <repository location="https://eclipse.baeyens.it/update/V4/nightly/" enabled="false" />
+      <repository location="https://eclipse.baeyens.it/update/V4/stable/" enabled="true" />
+      <repository location="https://download.eclipse.org/releases/latest/" enabled="true" />
       <repository location="https://download.eclipse.org/technology/babel/update-site/latest/" enabled="true" />
-      <repository location="http://https://download.eclipse.org/justj/jres/15/updates/release/latest/" enabled="true" />
       <repository location="https://download.eclipse.org/eclipse/updates/latest" enabled="true" />
+      <repository location="https://download.eclipse.org/justj/jres/17/updates/release/latest/" enabled="true" />
+      <repository location="https://download.eclipse.org/nebula/incubation/latest" enabled="true" />

```

Comment:
Should this not be removed or at least be false?

Reply:
Yes, I removed nebula incubation. Will also remove it from the feature.